(function(){"use strict";var t={389:function(t,e,a){var n=a(144),o=a(998),s=a(6843),i=a(6190),l=a(8323),r=a(8271),c=a(4324),p=a(3687),u=a(626),d=a(7045),m=function(){var t=this,e=t._self._c;t._self._setupProxy;return e(o.Z,[e(s.Z,{staticClass:"px-lg-16",attrs:{app:"",color:"secondary",dark:"","hide-on-scroll":""},scopedSlots:t._u([{key:"extension",fn:function(){return[e(d.Z,{attrs:{value:t.tabsSelected,"slider-size":"5",optional:""}},[e(u.Z,{staticClass:"pa-0 ma-0",staticStyle:{"min-width":"0px"}}),t._l(t.navItems,(function(a){return e(u.Z,{key:a.name,staticClass:"white--text font-weight-bold",attrs:{to:a.path}},[t._v(" "+t._s(a.name)+" ")])}))],2)]},proxy:!0}])},[e("h2",[e("router-link",{staticClass:"homepage-title",attrs:{to:"/"}},[t._v(" Unipept ")])],1),e(p.Z),e(i.Z,{attrs:{text:""}},[t._v(" Publications ")]),e("router-link",{attrs:{to:"/about"},scopedSlots:t._u([{key:"default",fn:function({href:a,navigate:n}){return[e(i.Z,{attrs:{text:"",href:a},on:{click:n}},[t._v(" About ")])]}}])})],1),e(l.Z,[e("router-view")],1),e(r.Z,{staticClass:"grey darken-3 px-lg-16 flex",attrs:{dark:"",padless:""}},[e("div",{staticClass:"d-flex flex-wrap",staticStyle:{width:"100%"}},[e("div",{staticClass:"col-md-8 text-md-start order-md-0 col-12 text-center order-2 grey--text d-md-flex"},[e("div",{staticClass:"flex-md-column"},[t._v(" "+t._s(t.$route.meta.publication)+" ")]),e("div",{staticClass:"flex-md-column"},[e("a",{staticClass:"link",attrs:{href:"https://"+t.$route.meta.publicationLink,target:"_blank"}},[t._v(" "+t._s(t.$route.meta.publicationLink))])])]),e("div",{staticClass:"col-md-4 text-md-end order-md-1 col-12 text-center order-0"},[e("a",{staticClass:"link",attrs:{href:"mailto:unipept@ugent.be"}},[e(c.Z,{staticClass:"grey--text link",attrs:{size:"30px"}},[t._v(" mdi-email-outline ")])],1),e("a",{staticClass:"ml-10 link",attrs:{href:"https://twitter.com/unipept",target:"_blank"}},[e(c.Z,{staticClass:"grey--text link",attrs:{size:"30px"}},[t._v(" mdi-twitter ")])],1),e("a",{staticClass:"ml-10 link",attrs:{href:"https://github.com/unipept/unipept",target:"_blank"}},[e(c.Z,{staticClass:"grey--text link",attrs:{size:"30px"}},[t._v(" mdi-github ")])],1)]),e("div",{staticClass:"col-md-8 text-md-start order-md-2 col-12 text-center order-3 grey--text"},[t._v("© 2022 Universiteit Gent")]),e("div",{staticClass:"col-md-4 text-md-end order-md-3 col-12 text-center order-1"},[e("router-link",{staticClass:"link",attrs:{to:"/about"}},[t._v("Terms of service")]),e("router-link",{staticClass:"link ml-5",attrs:{to:"/posts"}},[t._v("News")]),e("router-link",{staticClass:"link ml-5",attrs:{to:"/"}},[t._v("Publications")])],1)])])],1)},f=[],h=(0,n.defineComponent)({name:"App",data(){return{navItems:[{name:"Tryptic Peptide Analysis",path:"/tpa"},{name:"Metaproteomics Analysis",path:"/mpa"},{name:"API",path:"/apidocs"},{name:"CLI",path:"/clidocs"},{name:"Metagenomics",path:"/umgap"},{name:"Unipept Desktop",path:"/desktop"}]}}}),v=h,g=a(1001),_=(0,g.Z)(v,m,f,!1,null,null,null),b=_.exports,y=a(707),C=a.n(y);n["default"].use(C());var x=new(C())({theme:{themes:{light:{primary:"#2196F3",secondary:"#FFC107"}}}}),w=a(8345),k=a(579),P=a(4886),Z=a(266),S=a(2118),U=a(9223),T=a(1338),A=a(4525),L=a(6165),j=a(1713),I=function(){var t=this,e=t._self._c;t._self._setupProxy;return e(S.Z,[e(S.Z,{staticClass:"main"},[e(j.Z,[e(Z.Z,{staticClass:"col-lg-6 col-sm-12"},[e("div",{staticClass:"text-h3 font-weight-light mb-2"},[t._v("Welcome")]),t._v(" Unipept is an open source web application developed at "),e("a",{attrs:{href:"https://www.ugent.be/en",target:"_blank"}},[t._v("Ghent University")]),t._v(" that is designed for metaproteomics data analysis with a focus on "),e("span",{staticClass:"font-weight-bold"},[t._v("interactive datavisualizations")]),t._v(". Unipept is powered by an index containing all UniProt entries, a tweaked version of the NCBI taxonomy and a custom lowest common ancestor algorithm. This combination enables a blazingly fast "),e("span",{staticClass:"font-weight-bold"},[t._v("biodiversity analysis")]),t._v(" of large and complex "),e("span",{staticClass:"font-weight-bold"},[t._v("metaproteome samples")]),t._v(". This functionality is also available via an API and a set of command line tools. Next to these core functions, Unipept also has a tool for selecting unique peptides for "),e("span",{staticClass:"font-weight-bold"},[t._v("targeted proteomics")]),t._v(" and for "),e("span",{staticClass:"font-weight-bold"},[t._v("comparing genomes")]),t._v(" based on peptide similarity. ")]),e(Z.Z,{staticClass:"col-lg-6 col-sm-12"},[e(k.Z,[e(A.Z,{staticClass:"blue darken-1",attrs:{"two-line":""}},[e(L.km,[e(L.V9,{staticClass:"text-h5 white--text"},[t._v(" Unipept 4.3 ")]),e(L.oZ,{staticClass:"white--text"},[t._v("posted on 2020-04-15")])],1)],1),e(P.ZB,[t._v(" Unipept 4.3 has just been released and contains these major additions: "),e("ul",[e("li",[t._v(" Load and process multiple samples at once with the metaproteomics analysis tool. ")]),e("li",[t._v(" Compare multiple samples with each other using the new heatmap. ")]),e("li",[t._v(" Functional analysis has been expanded with support for InterPro annotations. ")]),e("li",[t._v(" UniProt updated to version 2020.01 ")])])]),e(U.Z,{staticClass:"mx-4"}),e(P.h7,[e(p.Z),e("router-link",{attrs:{to:"/posts"},scopedSlots:t._u([{key:"default",fn:function({href:a,navigate:n}){return[e(i.Z,{attrs:{color:"secondary",text:"",href:a},on:{click:n}},[t._v(" More news ")])]}}])})],1)],1)],1)],1),e(j.Z,[e(Z.Z,{staticClass:"col-lg-4 col-md-6"},[e("router-link",{attrs:{to:"/tpa"}},[e(T.Z,{scopedSlots:t._u([{key:"default",fn:function({hover:n}){return[e("HomePageCard",{attrs:{hover:n,title:"Tryptic Peptide Analysis",asset:a(6225)}},[t._v(" With tryptic peptide analysis, you can submit a single "),e("span",{staticClass:"font-weight-bold"},[t._v("tryptic peptide")]),t._v(" that can be 5 to 50 residues long. The application will respond with a list of "),e("span",{staticClass:"font-weight-bold"},[t._v("all Uniprot entries")]),t._v(" wherein the peptide was found along with a complete taxonomic lineage derived from the NCBI taxonomy. These lineages are presented as a comprehensible table and using an interactive tree view. ")])]}}])})],1)],1),e(Z.Z,{staticClass:"col-lg-4 col-md-6"},[e("router-link",{attrs:{to:"/mpa"}},[e(T.Z,{scopedSlots:t._u([{key:"default",fn:function({hover:n}){return[e("HomePageCard",{attrs:{hover:n,title:"Metaproteomics Analysis",asset:a(2307)}},[t._v(" Metaproteomics analysis helps you analyze "),e("span",{staticClass:"font-weight-bold"},[t._v("lists of tryptic peptides")]),t._v(", e.g. extracted from an environmental sample using shotgun tandem mass spectrometric methods. Of these peptides, the lowest common ancestors (LCA) will be calculated. These LCAs will be bundled and displayed in an interactive treemap giving you insight into the "),e("span",{staticClass:"font-weight-bold"},[t._v("biodiversity of your sample")]),t._v(". ")])]}}])})],1)],1),e(Z.Z,{staticClass:"col-lg-4 col-md-6"},[e("router-link",{attrs:{to:"/desktop"}},[e(T.Z,{scopedSlots:t._u([{key:"default",fn:function({hover:n}){return[e("HomePageCard",{attrs:{hover:n,title:"Unipept Desktop",asset:a(6633)}},[t._v(" The Unipept Desktop application allows to analyse larger samples, to store analysis results offline, to extensively compare assays with each other and to better manage samples. This application is powered by the Electron framework and runs on almost all systems. This allows you to improve the understanding of all interactions that are taking place in a complex environment. ")])]}}])})],1)],1),e(Z.Z,{staticClass:"col-lg-4 col-md-6"},[e("router-link",{attrs:{to:"/apidocs"}},[e(T.Z,{scopedSlots:t._u([{key:"default",fn:function({hover:n}){return[e("HomePageCard",{attrs:{hover:n,title:"API Documentation",asset:a(6294)}},[t._v(" Unipept offers most of its peptide analysis features as a "),e("span",{staticClass:"font-weight-bold"},[t._v("web service")]),t._v(". This enables the integration of Unipept functionality into other applications and the creation of batch processing scripts. These "),e("span",{staticClass:"font-weight-bold"},[t._v("documentation pages")]),t._v(" describe the available features of the API, how to access them and plenty examples. ")])]}}])})],1)],1),e(Z.Z,{staticClass:"col-lg-4 col-md-6"},[e("router-link",{attrs:{to:"/clidocs"}},[e(T.Z,{scopedSlots:t._u([{key:"default",fn:function({hover:n}){return[e("HomePageCard",{attrs:{hover:n,title:"CLI Documentation",asset:a(4686)}},[t._v(" The Unipept "),e("span",{staticClass:"font-weight-bold"},[t._v("command line interface")]),t._v(" (CLI) is a wrapper around the Unipept API that offers an easy way to "),e("span",{staticClass:"font-weight-bold"},[t._v("integrate Unipept")]),t._v(" functionality into your data processing "),e("span",{staticClass:"font-weight-bold"},[t._v("pipelines and scripts")]),t._v(". These pages cover installation and usage instructions, an overview of all available options and several use cases. ")])]}}])})],1)],1),e(Z.Z,{staticClass:"col-lg-4 col-md-6"},[e("router-link",{attrs:{to:"/umgap"}},[e(T.Z,{scopedSlots:t._u([{key:"default",fn:function({hover:n}){return[e("HomePageCard",{attrs:{hover:n,title:"UMGAP",asset:a(8810)}},[t._v(" Use the Unipept MetaGenomics Analysis Pipeline to assign taxonomic labels to your "),e("span",{staticClass:"font-weight-bold"},[t._v("shotgun metagenomics reads")]),t._v(". The results are available as taxonomic frequency tables and interactive visualizations. UMGAP is a collection of CLI tools that can be combined to identify shotgun metagenomics reads. ")])]}}])})],1)],1)],1)],1)],1)},M=[],O=a(5495),G=function(){var t=this,e=t._self._c;t._self._setupProxy;return e(k.Z,{staticClass:"equal",attrs:{elevation:t.hover?6:2}},[e(O.Z,{attrs:{height:"300",src:t.asset,alt:t.asset}}),e(P.EB,{staticClass:"feature-title"},[t._v(t._s(t.title))]),e(P.ZB,{staticClass:"mt-4"},[t._t("default")],2)],1)},H=[],z=(0,n.defineComponent)({name:"HomePageCard",setup(){},props:{title:String,asset:String,hover:Boolean}}),B=z,J=(0,g.Z)(B,G,H,!1,null,"4300042e",null),R=J.exports,D=(0,n.defineComponent)({name:"HomePage",setup(){},components:{HomePageCard:R}}),F=D,N=(0,g.Z)(F,I,M,!1,null,"6f62cfe8",null),$=N.exports,q=function(){var t=this,e=t._self._c;t._self._setupProxy;return e("div")},E=[],V=(0,n.defineComponent)({name:"AboutPage",setup(){}}),W=V,K=(0,g.Z)(W,q,E,!1,null,null,null),Q=K.exports,X=function(){var t=this,e=t._self._c;t._self._setupProxy;return e("div")},Y=[],tt=(0,n.defineComponent)({name:"NewsPage",setup(){}}),et=tt,at=(0,g.Z)(et,X,Y,!1,null,null,null),nt=at.exports,ot=function(){var t=this,e=t._self._c;t._self._setupProxy;return e("div")},st=[],it=(0,n.defineComponent)({name:"TrypticPeptideAnalysisPage",setup(){}}),lt=it,rt=(0,g.Z)(lt,ot,st,!1,null,null,null),ct=rt.exports,pt=function(){var t=this,e=t._self._c;t._self._setupProxy;return e("div")},ut=[],dt=(0,n.defineComponent)({name:"MetaproteomeAnalysisPage",setup(){}}),mt=dt,ft=(0,g.Z)(mt,pt,ut,!1,null,null,null),ht=ft.exports,vt=function(){var t=this,e=t._self._c;t._self._setupProxy;return e("div")},gt=[],_t=(0,n.defineComponent)({name:"APIPage",setup(){}}),bt=_t,yt=(0,g.Z)(bt,vt,gt,!1,null,null,null),Ct=yt.exports,xt=function(){var t=this,e=t._self._c;t._self._setupProxy;return e("div")},wt=[],kt=(0,n.defineComponent)({name:"CLIPage",setup(){}}),Pt=kt,Zt=(0,g.Z)(Pt,xt,wt,!1,null,null,null),St=Zt.exports,Ut=function(){var t=this,e=t._self._c;t._self._setupProxy;return e("div")},Tt=[],At=(0,n.defineComponent)({name:"MetagenomicsPage",setup(){}}),Lt=At,jt=(0,g.Z)(Lt,Ut,Tt,!1,null,null,null),It=jt.exports,Mt=function(){var t=this,e=t._self._c;t._self._setupProxy;return e("div")},Ot=[],Gt=(0,n.defineComponent)({name:"DesktopPage",setup(){}}),Ht=Gt,zt=(0,g.Z)(Ht,Mt,Ot,!1,null,null,null),Bt=zt.exports;n["default"].use(w.Z);const Jt=[{path:"/",component:$,meta:{publication:"Gurdeep Singh et al. (2019) Journal of Proteome Research",publicationLink:"doi:10.1021/acs.jproteome.8b00716"}},{path:"/tpa",component:ct,meta:{publication:"Gurdeep Singh et al. (2019) Journal of Proteome Research",publicationLink:"doi:10.1021/acs.jproteome.8b00716"}},{path:"/mpa",component:ht,meta:{publication:"Gurdeep Singh et al. (2019) Journal of Proteome Research",publicationLink:"doi:10.1021/acs.jproteome.8b00716"}},{path:"/apidocs",component:Ct,meta:{publication:"Mesuere et al. (2016) Bioinformatics",publicationLink:"doi:10.1093/bioinformatics/btw039"}},{path:"/clidocs",component:St,meta:{publication:"Verschaffelt et al. (2020) Bioinformatics",publicationLink:"doi.org/10.1093/bioinformatics/btaa553"}},{path:"/umgap",component:It,meta:{publication:"Gurdeep Singh et al. (2019) Journal of Proteome Research",publicationLink:"doi:10.1021/acs.jproteome.8b00716"}},{path:"/desktop",component:Bt,meta:{publication:"Verschaffelt et al. (2021) Journal of Proteome Research",publicationLink:"doi.org/10.1021/acs.jproteome.0c00855"}},{path:"/about",component:Q,meta:{publication:"Gurdeep Singh et al. (2019) Journal of Proteome Research",publicationLink:"doi:10.1021/acs.jproteome.8b00716"}},{path:"/posts",component:nt,meta:{publication:"Gurdeep Singh et al. (2019) Journal of Proteome Research",publicationLink:"doi:10.1021/acs.jproteome.8b00716"}}];var Rt=new w.Z({routes:Jt});n["default"].config.productionTip=!1,new n["default"]({vuetify:x,router:Rt,render:t=>t(b)}).$mount("#app")},6294:function(t,e,a){t.exports=a.p+"img/home-feature-api.1d211fe1.svg"},4686:function(t,e,a){t.exports=a.p+"img/home-feature-cli.033ed93d.svg"},2307:function(t,e,a){t.exports=a.p+"img/home-feature-multi.6cbde7dc.svg"},8810:function(t,e,a){t.exports=a.p+"img/home-feature-umgap.b7ccb1bd.svg"},6633:function(t,e,a){t.exports=a.p+"img/home-feature-desktop.f606e399.png"},6225:function(t,e,a){t.exports=a.p+"img/home-feature-single.85a7913c.png"}},e={};function a(n){var o=e[n];if(void 0!==o)return o.exports;var s=e[n]={exports:{}};return t[n].call(s.exports,s,s.exports,a),s.exports}a.m=t,function(){var t=[];a.O=function(e,n,o,s){if(!n){var i=1/0;for(p=0;p<t.length;p++){n=t[p][0],o=t[p][1],s=t[p][2];for(var l=!0,r=0;r<n.length;r++)(!1&s||i>=s)&&Object.keys(a.O).every((function(t){return a.O[t](n[r])}))?n.splice(r--,1):(l=!1,s<i&&(i=s));if(l){t.splice(p--,1);var c=o();void 0!==c&&(e=c)}}return e}s=s||0;for(var p=t.length;p>0&&t[p-1][2]>s;p--)t[p]=t[p-1];t[p]=[n,o,s]}}(),function(){a.n=function(t){var e=t&&t.__esModule?function(){return t["default"]}:function(){return t};return a.d(e,{a:e}),e}}(),function(){a.d=function(t,e){for(var n in e)a.o(e,n)&&!a.o(t,n)&&Object.defineProperty(t,n,{enumerable:!0,get:e[n]})}}(),function(){a.g=function(){if("object"===typeof globalThis)return globalThis;try{return this||new Function("return this")()}catch(t){if("object"===typeof window)return window}}()}(),function(){a.o=function(t,e){return Object.prototype.hasOwnProperty.call(t,e)}}(),function(){a.r=function(t){"undefined"!==typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(t,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(t,"__esModule",{value:!0})}}(),function(){a.p="/"}(),function(){var t={143:0};a.O.j=function(e){return 0===t[e]};var e=function(e,n){var o,s,i=n[0],l=n[1],r=n[2],c=0;if(i.some((function(e){return 0!==t[e]}))){for(o in l)a.o(l,o)&&(a.m[o]=l[o]);if(r)var p=r(a)}for(e&&e(n);c<i.length;c++)s=i[c],a.o(t,s)&&t[s]&&t[s][0](),t[s]=0;return a.O(p)},n=self["webpackChunkunipept_web"]=self["webpackChunkunipept_web"]||[];n.forEach(e.bind(null,0)),n.push=e.bind(null,n.push.bind(n))}();var n=a.O(void 0,[998],(function(){return a(389)}));n=a.O(n)})();
//# sourceMappingURL=app.05fd284f.js.map